index.js

```diff
@@ -534,29 +534,34 @@ app.post(
 app.use(braveAlerter.getRouter())
 
 // Handler for incoming XeThru POST requests
-app.post('/api/xethru', Validator.body(['locationid', 'state', 'rpm', 'mov_f', 'mov_s']).exists(), async (req, res) => {
+app.post('/api/xethru', Validator.body(['coreid', 'state', 'rpm', 'mov_f', 'mov_s']).exists(), async (req, res) => {
```

Comment:
Just wondering if this is actually breaking? On the one hand if you tried to set up a new radar with old instructions and didn't set the right coreID in the creation page, it wouldn't work, at the same time, it's still backwards compatible with the old firmware

Reply:
ah nvm, I just noticed in the clickup task that there's a webhook component to the change as well

Reply:
Yes, I've already update the production webhook to add the `coreid` (so that we don't get in trouble by forgetting to do it). Then we can take our time remembering to remove `locationid` from it. It looks like there were a couple of other things (like `device` in there that we don't use that we could clean up if we really wanted to.